README.md

```diff
@@ -154,6 +156,7 @@ and `{}` for a mutually exclusive keyword.
 | --- | --- | --- |
 | List databases | `SHOW DATABASES;` | |
 | Switch database | `USE <database>;` | |
+| Switch database with using roles | `USE <database> ROLE <role>;` | |
```

Comment:
Thank you for adding to README.

Could you add square brackets `[]` to denote that it's an optional keyword? For example: `USE <database> [ROLE <role>];`

Reply:
Sure. How about this? 8de7619

Reply:
LGTM!